tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -28,7 +28,7 @@ Verify Traefik Deployment
 Verify JH Deployment
     [Documentation]  Verifies RHODS JH deployment
     [Tags]    Sanity
-    ...       ODS-546  ODS-294
+    ...       ODS-546  ODS-294  ODS-1250
```

Comment:
Missing Tier1 tag

Reply:
Not Needed imho, it already has Sanity

Reply:
We'll discuss this in the next automation meeting